amida-tech/indaba-client
# Feature/inba 564 weighted questions

#### What does this PR do?
Allows dynamic question types (Checkboxes, Multiple Choice, Dropdowns) to have a weighted value entered.

#### Related JIRA tickets:
[INBA-564](https://jira.amida-tech.com/browse/INBA-564)

#### How should this be manually tested?
Go into a survey, whether new or pre-created. Enter three new questions: a Checkbox, a Multiple Choice and a Dropdown. For each, click on the "Weighted Value Question" checkbox. 

Enter in options, and weighted values. It is okay to leave weighted values blank if desired. Save the surveys.

Go to the `question_choice` table in surveyService. Ensure that the newly added choices have the entered values under the "weight" column.

#### Background/Context
There maybe issues because of inconsistencies on how survey is saved. 

#### Screenshots (if appropriate):
